app/lib/contentdm/record.rb

```diff
@@ -153,8 +153,19 @@ def remove_nils(property)
         property.select { |prop| !prop.nil? }
       end
 
+      ##
+      # @param values [Array]
+      # @return [Array]
       def strip_whitespace(values)
         values.map { |v| v.strip }
       end
+
+      ##
+      # @param title [String]
+      # @return [String]
+      def remove_author(title)
+        return title unless title.include?('/')
+        title.split('/')[0].strip! if title.include?('/')
```

Comment:
You don't need the `if title.include?('/')` on this line since you have it on the line above